### PR TITLE
fix: update responsiveness on large, medium, and small screens

### PR DESCRIPTION
**Pull request by willuhmjs**
Fixes issue #7 and more!

### CODE REVIEW

File: src/lib/Navbar.svelte [hunk 1]
```diff
@@ -38,16 +38,16 @@
...
 		color: inherit;
 		text-decoration: inherit;
-		margin-left: 16px;
-		margin-right: 16px;
+		margin-left: 1rem;
+		margin-right: 1rem;
 	}
 
-	@media screen and (max-width: 805px) {
+	@media screen and (max-width: 50.3125rem) {
```
Comment (tristan-f-r) [inline src/lib/Navbar.svelte]: This feels arbitrary

File: src/lib/Section/Section.svelte [hunk 2]
```diff
@@ -19,25 +19,27 @@
...
 
 	.section hr {
-		width: 570px;
-		margin-left: 0px;
-		margin-top: 20px;
-		height: 0.35px;
+		width: 35.625rem;
+		margin-left: 0rem;
+		margin-top: 1.25rem;
+		height: 0.021875rem;
```
Comment (tristan-f-r) [inline src/lib/Section/Section.svelte]: So does this - maybe it would be better to use fractions where applicable?

**Review by tristan-f-r — DISMISSED** [review]
There are a lot of subpixel values, and *some* of these should be corrected, as subpixel values don't make a visual difference. Otherwise, the rest of the changes look good :+1:

File: src/lib/Section/Section.svelte (hunk 2 above)
Comment (willuhmjs) [reply src/lib/Section/Section.svelte]: I agree that using fractions when the decimal is too long would make the code more readable. How about this SRS?
* All CSS values MUST be rounded to three decimal places.

Comment (tristan-f-r) [reply src/lib/Section/Section.svelte]: I don't know a rule in eslint that enforces that, and I would prefer to avoid manual style guides.

**Review by tristan-f-r — APPROVED** [review]
LGTM!

**Pull request merged**